markdown/preface.markdown

```diff
@@ -190,9 +194,8 @@ print >>sys.stderr, "fatal error"  # Python 2
 print("fatal error", file=sys.stderr)  # Python 3
 ```
 
-Ah, that looks a bit more worthwhile.
-What the hell is going on in that Python 2 statement?
-The authors don't rightly know.
+That change certainly seems worthwhile; and what is going on in the
+Python 2 version anyway?  The authors don't rightly know.
```

Comment:
Was Nan commenting on "The authors don't rightly know." in addition to removing "hell"? This is something Greg Wilson didn't like either. Maybe expand on this point a little. Why is this behaviour unexpected?